pkg/clearsource/clearsource.go

```diff
@@ -64,7 +65,11 @@ func BuildFn(ctx *gcp.Context, exclusions []string) error {
 		ctx.Span("Clear source", now, gcp.StatusOk)
 	}(time.Now())
 
+	userExclusions := strings.Split(strings.TrimSpace(os.Getenv(env.ClearSourceExclude)), ":")
```

Comment:
I don't think we should be trimming space here. Spaces are allowed path characters, right?

Reply:
We're trimming spaces at the beginning and end of the string to sanitize environment input.

Reply:
I understand that, but spaces are an allowed file/directory name character, as unlikely as it is that someone would have a trailing space at the end of a directory.

Reply:
Understood, will remote TrimSpace here.